docs/MigrationGuide.md

```diff
@@ -357,6 +358,7 @@ The following changes were made to the `new` family of functions in the C API:
 
 - `ecs_new(world, T)` got renamed to `ecs_new_w`
 - `ecs_new_id` got renamed to `ecs_new`
+- `ecs_new_entity(world, "EntityName")` got renamed to `ecs_entity(world, { .name = "EntityName" })`
```

Comment:
Change to: "`ecs_new_entity` no longer exists, use `ecs_entity(world, { .name = "EntityName" })` instead"